jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Session.java

```diff
@@ -1470,6 +1480,14 @@ default void onClose(Session session)
         }
     }
 
+    /**
+     * <p>Exception indicating that the HTTP/2 stream was not handled
+     * by the server, and can therefore be retried, if possible.</p>
+     */
+    public static class RetryStreamException extends RuntimeException
```

Comment:
This doesn't need to be a `RuntimeException`, so it could extend from `Exception`.